components/automate-ui/src/app/pages/automate-settings/automate-settings.component.spec.ts

```diff
@@ -63,21 +86,40 @@ describe('AutomateSettingsComponent', () => {
   it('sets defaults for all form groups', () => {
     expect(component.automateSettingsForm).not.toEqual(null);
     expect(component.automateSettingsForm instanceof FormGroup).toBe(true);
-    expect(Object.keys(component.automateSettingsForm.controls)).toEqual([
-      'eventFeed',
-      'clientRuns',
-      'complianceData',
-      'missingNodes',
-      'deleteMissingNodes'
-    ]);
+    expect(Object.keys(component.automateSettingsForm.controls)).toEqual(ALL_FORMS);
   });
 
-  describe('patchDisableValue(form, value)', () => {
-    it('updates the value of the disable control from the provided form', () => {
-      expect(component.clientRunsForm.value.disable).toEqual(false);
-      component.patchDisableValue(component.clientRunsForm, true);
-      expect(component.clientRunsForm.value.disable).toEqual(true);
+  describe('handleFormActivation(form, value)', () => {
+
+    using(ALL_FORMS
+        // Service Groups on not currently uncheckable through the UI
```

Comment:
```suggestion
        // Service Groups are not currently uncheckable through the UI
```
Same applies in next `using` block below.